src/Sylius/Bundle/ApiBundle/spec/Serializer/ProductVariantNormalizerSpec.php

```diff
@@ -38,8 +42,21 @@ function it_supports_only_product_variant_interface(ProductVariantInterface $var
         $this->supportsNormalization($order)->shouldReturn(false);
     }
 
-    function it_does_not_support_if_item_operation_name_is_admin_get(ProductVariantInterface $variant): void
-    {
+    function it_supports_normalization_if_section_is_not_admin_get(
+        ProductVariantInterface $variant,
+        SectionProviderInterface $sectionProvider,
+        ShopApiSection $shopApiSection
+    ): void {
+        $sectionProvider->getSection()->willReturn($shopApiSection);
+        $this->supportsNormalization($variant, null, ['item_operation_name' => 'shop_get'])->shouldReturn(true);
+    }
+
+    function it_does_not_support_if_section_is_admin_get(
+        ProductVariantInterface $variant,
+        SectionProviderInterface $sectionProvider,
+        AdminApiSection $adminApiSection
+    ): void {
+        $sectionProvider->getSection()->willReturn($adminApiSection);
         $this->supportsNormalization($variant, null, ['item_operation_name' => 'admin_get'])->shouldReturn(false);
```

Comment:
I believe `null, ['item_operation_name' => '...']` parameters are not required, here and above 🖖